content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/timeline.svg

```diff
@@ -0,0 +1,2008 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```

Comment:
You have the `timeline.drawio` source file, please change those English word to Mandarin and export again.